app/models/payment.rb

```diff
@@ -189,6 +190,21 @@ def sync_with_payout_processor
     sync_with_paypal if processor == PayoutProcessorType::PAYPAL
   end
 
+  def payout_failure_reason
+    return nil unless failure_reason.present?
+
+    puts "Payout failure reason: #{failure_reason}"
+    if processor == PayoutProcessorType::PAYPAL
+      reason = PAYPAL_FAILURE_SOLUTIONS[failure_reason]&.dig(:reason)
+    elsif processor == PayoutProcessorType::STRIPE
+      reason = STRIPE_FAILURE_SOLUTIONS[failure_reason]&.dig(:reason)
+    else
+      reason = failure_reason
```

Comment:
Don't think we want to directly display the `failure_reason` (which could be an error code) here. If there's no `failure_reason` or a corresponding error message is not found, we should display a generic error message like `We were unable to process this payout`.